src/Pub/API/Controllers/ProjectsController.cs

```diff
@@ -0,0 +1,58 @@
+using Domain.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+
+namespace API.Controllers
+{
+    /// <summary>
+    ///  This controller class defines the endpoints
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProjectsController
+    {
+        public ActionResult<IEnumerable<Project>> GetProjects()
```

Comment:
```suggestion
        public ActionResult<IEnumerable<ProjectDto>> GetProjects()
```

Reply:
Let's update the return type here to `ProjectDto` - the service/application will only communicate to the outside world using the dto objects.